pdf.go

```diff
@@ -859,6 +859,45 @@ func (w *pdfPageWriter) EndTextObject() {
 	w.inTextObject = false
 }
 
+/* Extract from the pdf specification:
```

Comment:
In general I think the convention is to use single line comments (spread over multiple lines) here, i.e. `// comment ...` etc.  The `stringBytesEscaper` is not exported, so this comment won't show up in the godocs. I would probably name this `escaper`, because there is no usage of `string` and `bytes` is pretty evident.

Reply:
Good point